sklearn/metrics/pairwise.py

```diff
@@ -1062,7 +1073,8 @@ def sigmoid_kernel(X, Y=None, gamma=None, coef0=1):
     return K
 
 
-def rbf_kernel(X, Y=None, gamma=None):
+@_deprecate_positional_args
+def rbf_kernel(X, Y=None, *, gamma=None):
```

Comment:
if I were you I'd leave this and other kernels with few parameters alone. Likely to break code for no good reason.

Reply:
ok